projects/core/src/lib/datasets/upload/upload.component.html

```diff
@@ -212,7 +212,14 @@
             <p>
                 <mat-form-field appearance="fill">
                     <mat-label>Dataset Name</mat-label>
-                    <input matInput type="text" placeholder="New Dataset" formControlName="name" />
+                    <input matInput type="text" placeholder="data_set_name" formControlName="name" />
```

Comment:
maybe it would make sense to have the user input the display name first and then aumatically generate a valid name for it, which they can then optionally adjust?

Reply:
[350445f](https://github.com/geo-engine/geoengine-ui/pull/624/commits/350445fa7744018cc434bf2c67cbfc67ad275916)